indico/htdocs/js/indico/modules/surveys.js

```diff
@@ -53,6 +53,62 @@
         $('.js-add-question-dropdown').dropdown();
     }
 
+    global.setupSubmissionButtons = function setupSubmissionButtons() {
+        $('.submission-ids').click(function(ev) {
+            var $this = $(this);
+            if ($this.prop('checked')) {
+                $('#export-submissions').text($T('Export selected submissions'));
+            } else {
+                var filterFn = function () {
+                    return $(this).prop('checked');
+                };
+
+                if ($('.submission-ids').filter(filterFn).length == 0) {
+                    $('#export-submissions').text($T('Export all submissions'));
+                }
+            }
+        });
+
+        $('#export-submissions').click(function(evt) {
```

Comment:
i think the jquery docs recomment using the proper methods over the shortcuts somewhere